Move logging properties on values.yaml

flavio-assis opened the pull request:
Since `remote_logging` and `logging_level` was under  `core` section on config, it was throwing deprecation messages on the Pods logs. I moved it into the `logging` section

<!--
Thank you for contributing! Please make sure that your code changes
are covered with tests. And in case of new features or big changes
remember to adjust the documentation.

Feel free to ping committers for the review!

In case of existing issue, reference it using one of the following:

closes: #ISSUE
related: #ISSUE

How to write a good git commit message:
http://chris.beams.io/posts/git-commit/
-->

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

flavio-assis committed:
Mode logging properties on values.yaml

Since `remote_logging` and `logging_level` was under  `core` section on config, it was throwing deprecation messages on the Pods logs. I moved it into the `logging` section

flavio-assis committed:
Trim whitespace

mik-laj commented on chart/values.yaml:
```diff
@@ -644,13 +644,13 @@ config:
   api:
     auth_backend: airflow.api.auth.backend.deny_all
   logging:
+    remote_logging: '{{- ternary "True" "False" .Values.elasticsearch.enabled }}'
```
Where possible, we try to maintain compatibility with Airflow 1.10 and 2.0 at the same time. I think we need to duplicate these keys. See: https://github.com/apache/airflow/blob/833c53b6a4ebd580c0e5daf75215d38e875d5ddd/chart/values.yaml#L668

flavio-assis replied:
Done!

flavio-assis committed:
Add airflow 1.10 compatibility

Add: 
`{{ .Values.config.core.colored_console_log }}` and
`{{ .Values.config.core.remote_logging }}`
for Airflow 1.10 compatibility

flavio-assis committed:
Fix tab identation
